### PR TITLE
Create portal demo image

### PR DESCRIPTION
**Pull request by slovit**

### CODE REVIEW

File: docker/portal-demo/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# License
```
Comment (btiernay) [inline docker/portal-demo/Dockerfile]: Banner: https://github.com/icgc/icgc-get/blob/develop/Dockerfile

File: docker/portal-demo/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  sed -i.bak '2iunset MAIL' $DCC_HOME/dcc-portal-server/bin/dcc-portal-server && \
+  rm $DCC_HOME/dcc-portal-server/bin/dcc-portal-server.bak
+COPY portal-application.yml $DCC_HOME/dcc-portal-server/conf/application.yml
+
+#
+# Post-installation
+#
+RUN \ 
+  chown -R $DCC_USER:$DCC_GROUP $DCC_HOME && \
+  rm -r /var/log/elasticsearch /tmp/* /var/lib/apt/lists/*
```
Comment (btiernay) [inline docker/portal-demo/Dockerfile:74]: No entrypoint?

File: docker/portal-demo/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+#
+# Configuration
+#
+ENV DCC_HOME /opt/dcc
+ENV DCC_USER dcc
+ENV DCC_GROUP dcc
+RUN useradd -m $DCC_USER &&  mkdir -p $DCC_HOME
+COPY import_data.sh $DCC_HOME/import_data.sh
+ENTRYPOINT ["/opt/dcc/import_data.sh"]
```
Comment (btiernay) [inline docker/portal-demo/Dockerfile]: Ah

Comment (btiernay) [inline docker/portal-demo/Dockerfile]: Use to this being at the bottom. Any reason you put it here? For layer reuse?

Comment (slovit) [inline docker/portal-demo/Dockerfile]: Didn't know it should be at the bottom.

File: docker/portal-demo/Dockerfile [hunk 4]
```diff
@@ -1,4 +1,10 @@
-# License 
+#     ____  ____________   ____             __        __   ____                     
+#    / __ \/ ____/ ____/  / __ \____  _____/ /_____ _/ /  / __ \___  ____ ___  ____ 
+#   / / / / /   / /      / /_/ / __ \/ ___/ __/ __ `/ /  / / / / _ \/ __ `__ \/ __ \
+#  / /_/ / /___/ /___   / ____/ /_/ / /  / /_/ /_/ / /  / /_/ /  __/ / / / / / /_/ /
+# /_____/\____/\____/  /_/    \____/_/   \__/\__,_/_/  /_____/\___/_/ /_/ /_/\____/ 
+#                                                                                  
+# Banner @ http://goo.gl/VCY0tD
```
Comment (btiernay) [inline docker/portal-demo/Dockerfile]: http://goo.gl/WVEzIZ

File: docker/portal-demo/README.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
+ICGC DCC - Portal Demo
+===
```
Comment (btiernay) [inline docker/portal-demo/README.md]: Please always use `#` instead. Thanks.

File: docker/portal-demo/README.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+ICGC DCC Portal Demo is a docker container that allows to configure and run a standalone version of the [ICGC DCC Portal.](https://dcc.icgc.org/)
```
Comment (btiernay) [inline docker/portal-demo/README.md:4]: Please mention how this is for demo purposes only and not intended for production environments.

**Pull request merged**